Makefile

```diff
@@ -47,7 +47,7 @@ GOLANGCI_COMPOSER_IMAGE=composer_golangci
 #         pre-fetched but evaluated at time of use.
 #
 
-VERSION := $(shell (cd "$(SRCDIR)" && grep "^Version:" image-builder-cli.spec | sed -n 's/^[^0-9]*\([1-9][0-9]*\(\.[1-9][0-9]*\)*\)/\1/p'))
+VERSION := $(shell ( git describe --tags --abbrev=0 2>/dev/null || echo v1 ) | sed 's|v||')
```

Comment:
I don't think that this is the correct fix.

When I look at the osbuild repository, `setup.py` has version `138`, SPEC file has `138` and the latest tag is `v137`.

Makefile uses https://github.com/osbuild/osbuild/blob/fd19ab41fb5a84a15560ca319d2dab4f59c06e27/Makefile#L50

And packit config contains exactly the same way to determine the current version as the one in this repository.

TL;DR; I think we should keep using the Version from the SPEC, which is the next "to be released" version.

Reply:
OK, I'm probably wrong, since from looking at osbuild COPR build logs for commits on `main`, I can see that the `package_version` is based on the last tag https://download.copr.fedorainfracloud.org/results/@osbuild/osbuild/fedora-41-x86_64/08476002-osbuild/builder-live.log.gz

The problem is probably that since Packit does not control tarball generation in this case, it can't control the version in its filename.